website/types/env.d.ts

```diff
@@ -9,6 +9,7 @@ declare global {
       ADMIN_USERS: string;
       MODERATOR_USERS: string;
       INFERENCE_SERVER_HOST: string;
+      ENABLE_CHAT: boolean;
```

Comment:
this should be string, after all, are environment variables are strings.

Reply:
Then we need to remove all the `boolean(...)` from our code because it is redundant?

https://github.com/LAION-AI/Open-Assistant/blob/c67e1962904c895d6f9a0e7ba8c430aebe39623c/website/src/pages/api/chat/message.ts#L6

But that would not work, right?